README.md

```diff
@@ -120,6 +120,10 @@ null_ls.setup({
 > [!NOTE]
 > This only works when `nixfmt` is available [in the environment](#in-the-environment).
 
+#### Visual Studio Code (VSC)
+
+[vscode-nix-ide](https://github.com/nix-community/vscode-nix-ide) can invoke `nixfmt`.
+
```

Comment:
```suggestion

> This only works when `nixfmt` is available [in the environment](#in-the-environment).
```

I assume this note that's in many of the other in-editor examples, also applies to vscode-nix-ide?